.github/workflows/test.yml

```diff
@@ -58,3 +57,16 @@ jobs:
         run: tox
         env:
           DATABASE_TYPE: ${{ matrix.database-type }}
+          RUN_COVERAGE: ${{ matrix.run-coverage }}
+
+      - name: Coverage report
+        if: matrix.run-coverage == 'yes'
+        run: tox -e cov-report
+
+      - name: 'Upload coverage'
+        if: matrix.run-coverage == 'yes'
+        uses: actions/upload-artifact@v3
+        with:
+          name: factory_coverage_${{ matrix.python-version[1] }}_${{ matrix.database-type }}
```

Comment:
Would be nice to move this over to a codecov / equivalent and not have to download it..